src/pages/Post/index.tsx

```diff
@@ -49,10 +49,11 @@ const Post = () => {
     return (
         <BoxBase
             display="flex"
-            bgcolor="primary"
+            bgcolor="common"
             style={{ minHeight: "100vh" }}
             justifyContent="flex-end"
             flexDirection="column"
+            paddingTop={10}
```

Comment:
I think we should let its parent decide this padding

Reply:
![Screen Shot 2022-01-16 at 22 49 29](https://user-images.githubusercontent.com/59289834/149667158-3e5ebda6-df32-4ca1-b10b-4d9ebf3c85b8.png)
Without padding top our UI look like this

Reply:
If we use parent component to decide the padding it will look like this
![Screen Shot 2022-01-16 at 22 52 12](https://user-images.githubusercontent.com/59289834/149667245-98a4e86f-574f-46b9-be65-a7809c0f767e.png)

Reply:
ok